docs/tools/markdown-it-rule-example.js

```diff
@@ -73,7 +73,7 @@ function markdownItRuleExample({ open, close }) {
 			}
 
 			const { type, languageOptionsJSON } =
-				/^\s*(?<type>\S+)(\s+(?<languageOptionsJSON>\S.*?))?\s*$/u.exec(
+				/^\s*(?<type>\S+)(?:\s+(?<languageOptionsJSON>\S(?:.*\S)?))?\s*$/u.exec(
```

Comment:
```suggestion
				/^\s*(?<type>\S+)(?:\s+(?<languageOptionsJSON>\S(?:.*\S)?))?/u.exec(
```

The `\s*$` part is no longer needed now?